wordpool/__init__.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import pandas as pd
 from pkg_resources import resource_filename, resource_listdir
+from init_no_pandas import *
```

Comment:
This should use either an explicit relative import (`from .init_no_pandas ...`) or an absolute import (`from wordpool.init_no_pandas ...`). I would also prefer keeping with PEP8 module naming conventions (e.g., call the module `nopandas` or think of a better existing place to put the function).